### PR TITLE
Fixed bug in IE IR Reader

### PR DESCRIPTION
**Pull request by vgavrilo**
Fixed bug in IE IR Reader.

### CODE REVIEW

**Comment by lazarevevgeny** [comment]
@vgavrilo , please, add a IR Reader test for Interpolate-1 and Interpolate-4 to "inference-engine/tests/functional/inference_engine/ngraph_reader/".

File: inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp [hunk 1]
```diff
@@ -1711,126 +1711,6 @@ CNNLayer::Ptr NodeConverter<ngraph::op::v0::Interpolate>::createLayer(const std:
     THROW_IE_EXCEPTION << "Interpolate operation should be converted to Interp";
 }
 
-template <>
-CNNLayer::Ptr NodeConverter<ngraph::op::v4::Interpolate>::createLayer(const std::shared_ptr<ngraph::Node>& layer) const {
```
Comment (lazarevevgeny) [inline inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp:1715]: @vgavrilo , you removed correct code. This function should exist. You must remove LayerCreator for Interpolate-1 and Interpolate-4 in the inference-engine/src/readers/ir_reader/ie_ir_parser.cpp.

Comment (vgavrilo) [reply inference-engine/src/legacy_api/src/ie_cnn_layer_builder_ngraph.cpp]: Fixed.

**Comment by lazarevevgeny** [comment]
Fixed as part of PR #2051

**Pull request closed without merging**